README.md

```diff
@@ -49,27 +49,33 @@ Introspection
 
 SwiftUI | UIKit | AppKit | Introspect
 --- | --- | --- | ---
-List | UITableView | NSTableView | `.introspectTableView()`
-ScrollView | UIScrollView | NSScrollView | `.introspectScrollView()`
-NavigationView | UINavigationController | _N/A_ | `.introspectNavigationController()`
+NavigationView (StackNavigationViewStyle) | UINavigationController | _N/A_ | `.introspectNavigationController()`
+NavigationView (DoubleColumnNavigationViewStyle) | UISplitViewController | N/A | `.introspectSplitViewController()`
```

Comment:
Please use the same styling for `N/A` everywhere.
Sometimes it's `_N/A_` and sometimes it's `N/A`.

Reply:
Done